### DISCUSSION TITLE
Upgrade Crashlytics SDK

### DISCUSSION THREAD

**Discussion by Suraj3505** [invertase/react-native-firebase#4230] [Q&A] [answered]
Hi,

I want to upgrade crashlytics SDK as old SDKs are no longer supported. How can I upgrade crashlytics sdk without migrating to v6?

Thank you

**Comment by TommyLeong** [comment]
Don't think this is possible actually. I recall in v6 onwards, Firebase Crashlytics SDK is being use. [Bear in mind](https://firebase.googleblog.com/2020/06/crashlytics-sdk-now-available.html)..
> Fabric SDK will continue reporting your app's crashes until November 15, 2020

**Comment by mikehardy** [comment, marked as answer]
@Suraj3505 v5 will receive no support and will not receive a backport of the Firebase Crashlytics change. You can attempt the backport yourself using the PR against v6+ for guidance if you like but if there is one thing I can emphasize strongly: you need to update to the current stable versions.

We integrated about 15 user-facing functionality improvements or fixes **just this month** https://github.com/invertase/react-native-firebase/pulse/monthly and one of those was to bump the underying library versions which pulls in another pile of fixes (https://github.com/firebase/firebase-android-sdk/pulse/monthly / https://github.com/firebase/firebase-ios-sdk/pulse/monthly) - and a lot of those are targetted for instance at working on ios14 which will release shortly.

Why would you not make the necessary changes (migrating to v6) so that you can absorb these changes? It's been more than a year since the v5/v6 switch: it's time

**Reply by ANIBIT14** [reply]
Hi @mikehardy the main problem for us in migration this app of our's from v5 to v6 is loss of notifications support, as this app is already serving more than 45K customers daily, migrating to v6 is not an issue, but integrating notifee will take time on our end for this app, if their is no other way for updation of crashlytics sdk, then we will surely move to v6.

**Reply by mikehardy** [reply]
There was some discussion of being able to integrate just the v6+ version of crashlytics while still having v5 for everything else but this is nothing I'd recommend or support as a maintainer for understandable (I hope). As a programmer I have done terrible-but-necessary things before so I'd understand, we just can't support it. My fear as a programmer is that if you try this and anything goes wrong you've spent maybe the same amount of time needed to forward port. It looks like it is pretty finicky to make it work https://github.com/invertase/react-native-firebase/issues/3510#issuecomment-703563394 (though maybe using 'Core' again vs 'CoreOnly' could do it in the podspec - who knows)

**Reply by ANIBIT14** [reply]
Thanks @mikehardy, we have decided to migrate this app of our's to react-native-firebase v6,
